README.md

```diff
@@ -1,3 +1,4 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15535780&assignment_repo_type=AssignmentRepo)
 # SE_Day1
```

Comment:
The key difference between Agile vs. Waterfall is that Waterfall breaks down software development into isolated phases that flow into each other, while Agile advocates iterative development cycles in which multiple lifecycle phases can run in parallel.

How do Waterfall and Agile compare? Agile practitioners know that in the world of software development, things never go according to plan. Therefore, the impetus to continuously delivery software into the hands of the client is arguably the primary difference between Waterfall and Agile.

Waterfall asserts that customer feedback and stakeholder interactions need not occur until the end-product is delivered. However, Agile practitioners believe the following:

Requirements can become outdated before development even begins. New technologies can force changes to even the best designs. Wireframes and visual renderings don't always translate well into webpages and mobile apps. Revised stakeholder priorities can shift development timelines and efforts. Waterfall does not deal with these situations well.